docsets/Salesforce_Visualforce/docset.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "Salesforce Visualforce",
-    "version": "208.0",
+    "version": "210.0",
     "archive": "Salesforce_Pages.tgz",
```

Comment:
This archive wasn't updated. You added `Salesforce_Visualforce.tgz` instead.

Reply:
Renamed. Thanks.